### ISSUE TITLE
Validate test set 

### ISSUE THREAD

**Issue by adelchellabi** [ultralytics/yolov5#8124]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello 
My dataset is divided to three sets (train ,validation, test )
I want to do a validation for my test set (not the validation set ) to get the accuracy,precision ,etc ...
Please how to do it because when i use this command 
`python val.py --weights best.py --data data.yaml --task test `
I got x labels missing 
Is it necessary to do an annotation for the test set ?
Or how to validate the test set

**Comment by glenn-jocher** [comment]
@adelchellabi obviously metrics can only be computed on labelled data

**Comment by adelchellabi** [comment]
@glenn-jocher  thank you very much 😊

**Comment by glenn-jocher** [comment]
You're welcome, @adelchellabi! Feel free to reach out if you have any more questions. 😊
